lib/build-cron.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+const cmaster = require('cron-master');
+const log = require('./log')();
+const VError = require('verror');
+const timestring = require('timestring');
+const moment = require('moment');
+const getEventHandlers = require('./event-handlers');
+
+/**
+ * Grabs the Cron Job config files in the specified directory (cronConfigsDir), initiates and
+ * starts them
+ * @param {String} cronConfigsDir - complete path to config files for cron-master Cron Jobs
+ * @param {String} collection - Name of Collection to store Cron Logs in
+ * @param {Object} logCleanUpOpts - Options for Cleaning Up Cron Job Logs
+ * @return {Object}
+ */
+module.exports = function buildCronJobs(cronConfigsDir, collection, logCleanUpOpts) {
+
+  logCleanUpOpts =  logCleanUpOpts || {};
+
+  if (!cronConfigsDir || !collection) {
+    throw new VError('Error: \'cronConfigsDir\' and \'collection\' are required params.');
+  }
+
+  const db = require('rhmap-mongodb').collection(collection);
```

Comment:
Something that's kind of irked me that could probably be fixed here: the name `db` is not really correct. I think we should rename the `collection` argument to `collectionName` and then this can be renamed to `collection`. (The argument name in getHandlers() is already "collection".)